app/modules/site_settings/resources.py

```diff
@@ -352,7 +352,10 @@ def post(self, path):
             elif path in SiteSetting.get_setting_keys():
                 if '_value' not in data.keys():
                     abort(400, 'Need _value as the key in the data setting')
-                SiteSetting.set_key_value(path, data['_value'])
+                if data['_value']:
```

Comment:
in slack i see yesterday me talking about this -- as if i left a comment here.  but obviously i didnt!  i think this conditional needs to be adjusted to allow for the (new) `boolean` column/type SiteSetting has now.  namely, a user passing in a `False` (to set the key with that value) will end up getting `forget_key_value()` instead.   i _believe_ `is None` might do the trick?  (tho my python powers with evaluation false-ish things is not always what i think it is.)

Reply:
NO you're right and I missed that. Will push change first thing tomorrow my time.